src/main/java/com/quizapp/enterprise/controllers/GameController.java

```diff
@@ -27,7 +27,7 @@
     SimpMessagingTemplate messagingTemplate;
 
 
-    @PostMapping(value = "/newGame/{quizId}")
+    @PostMapping(value = "/new-game/{quizId}")
```

Comment:
URLs/URIs should use `kebab-case` instead of `camelCase`

https://medium.com/@victor.leong.17/5-url-best-practices-936329ba36a7